lms/templates/courseware/courseware.html

```diff
@@ -234,6 +234,25 @@
     </div>
 % endif
     <section class="course-content" id="course-content">
```

Comment:
@cahrens -- did a bit of research into this one -- UNDEFINED is the default value for a missing parameter on a mako template -- see here: http://docs.makotemplates.org/en/latest/runtime.html#context-variables